nuxeo-tree-snapshot-jsf/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 1
+Bundle-Name: Nuxeo ECM Snapshot Manager
+Bundle-SymbolicName: org.nuxeo.snapshot;singleton:=true
```

Comment:
You must change the symbolic name to be different than the core one. For instance `org.nuxeo.snapshot.jsf`.